app/views/account/your-details/subjects.html

```diff
@@ -43,12 +43,6 @@ <h1 class="govuk-heading-l">{{ subHeading }}</h1>
       <section class="govuk-!-margin-bottom-8">
         {% include "_includes/summary-cards/subject-status.html" %}
       </section>
-      <div class="govuk-button-group">
-        {{ govukButton({
-          text: "Request changes",
-          href: "/change-subjects"
```

Comment:
For this ticket we can also remove the '/change subjects' route in routes.js file.

Reply:
Ok, i've just removed it